README.md

```diff
@@ -62,13 +62,15 @@ Additional configuration options supported: https://github.com/axios/axios#reque
 
 Returns a promise which resolves, if successful, to an instance of `MetadataReader`.
 
-### toPassportConfig(reader)
+### toPassportConfig(reader, options = { multipleCerts: false })
 
 Transforms metadata extracts for use in Passport strategy configuration. The following strategies are currently supported:
 
 * [passport-saml](http://npmjs.org/packages/passport-saml)
 * [passport-wsfed-saml2](http://npmjs.org/packages/passport-wsfed-saml2)
 
+When setting the `multipleCerts` options to true, an array of signing certificates are passed to the passport config instead of assuming the last certificate is the most recent one.
```

Comment:
Please add a note indicating that this is only compatible with `passport-saml` and is not compatible with `passport-wsfed-saml2`.